oresat_configs/_yaml_to_od.py

```diff
@@ -672,7 +672,7 @@ def _gen_c3_beacon_defs(c3_od: canopen.ObjectDictionary, beacon_def: BeaconConfi
     return beacon_objs
 
 
-def _gen_fw_base_od(oresat_id: OreSatId, config_path: str) -> canopen.ObjectDictionary:
+def _gen_fw_base_od(oresat: Consts, config_path: str) -> canopen.ObjectDictionary:
```

Comment:
Same deal as earlier, please either add context to the var name or add a docstring describing it.